src/ResourceManager/Sql/Commands.Sql/packages.config

```diff
@@ -1,5 +1,11 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Azure.Management.Sql" version="1.10.0-preview" targetFramework="net452" />
```

Comment:
you can remove any dependencies that appear in common.dependencies.targets: https://github.com/Azure/azure-powershell/blob/preview/tools/Common.Dependencies.targets from both here and csproj.

Reply:
Done.